Services/Models/DeviceModel.cs

```diff
@@ -19,30 +20,36 @@ public class Script
 
         public Script()
         {
-            this.Type = "javascript";
+            this.Type = ScriptInterpreter.JAVASCRIPT_SCRIPT;
             this.Path = "scripts" + System.IO.Path.DirectorySeparatorChar;
             this.Params = null;
         }
     }
 
     public class DeviceModel
     {
+        public string ETag { get; set; }
```

Comment:
add `[JsonIgnore]` so that this property doesn't end into the storage record payload (we use the storage record ETag for this property)

Reply:
Done.